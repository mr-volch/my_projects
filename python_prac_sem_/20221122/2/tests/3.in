hello my 12345 friend ðóñàê
