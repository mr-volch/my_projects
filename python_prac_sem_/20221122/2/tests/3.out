hello my 12345 friend русак

